src/components/Footer/FooterItem.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import Link from "next/link";
+// import footerStyle from "./Footer.style.scss";
+
+function FooterItem() {
+  return (
+    <ListItem className="">
+      <ListItemText className="footerItemText">
```

Comment:
Please store all Items values in Footer.config.js file - like JSON. 
Array with objects, every object, has two fields: name and link. Make Link value same as Name (spaces replace with dash)

Reply:
How can I insert content of Footer.config.js in the FooterItem file? By using filter method on every MenuItem?
Seems unnecessary complicated.

Reply:
How I mentioned earlier, I think you don't need FooterItem at all.
You can do all iteration in Footer.component.js
Get JSON from Footer.config.js (array of items)
Map over them to render MenuItems from MUI + Link component from Next.js